algorithms/src/Kokkos_Sort.hpp

```diff
@@ -628,6 +644,17 @@ std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
   bin_sort.sort(exec, view);
 }
 
+#if defined(KOKKOS_ENABLE_CUDA)
+template <class DataType, class... Properties>
+void sort(const Cuda& space, const Kokkos::View<DataType, Properties...>& view,
+          bool const = false) {
```

Comment:
Could you give the name for `always_use_kokkos_sort` here?

Is there actually a way to call the non-Thrust implementation through the `Kokkos::sort` overload set on these arguments? It seems that the specialization for `Cuda` as `ExecutionSpace` would always win.

We may need to tweak the template parameters to expect a tag like `enum class SORT_IMPLEMENTATION { default, kokkos, backend_library };` and `enable_if` or `if constexpr` on it.

Reply:
Or, at least if `always_use_kokkos_sort` is passed `true`, then call `Kokkos::sort<Cuda, decltype(View)>(space, view, true)` to get our generic implementation?

Reply:
Why do we have this argument in the first place?

Reply:
Wasn't the idea to have a distinct function for CUDA (and other exec_spaces) View? Without any fallback? If that's not the case, then I'll rework this.

Reply:
I am saying I expect the signature would be
```C++
template <class DataType, class... Properties>
void sort(const Cuda& space, const Kokkos::View<DataType, Properties...>& view);

Reply:
@PhilMiller @dalg24 
Here https://github.com/kokkos/kokkos/pull/5372 I removed that argument completely and extracted std::sort to separated overload for serial space. If that PR will be accepted, then I can get back here to CUDA overload.

Reply:
That's resolved in https://github.com/kokkos/kokkos/pull/5388 and https://github.com/kokkos/kokkos/pull/5372